modules/apps/app-builder/app-builder-test/src/testFunctional/macros/AppBuilderAdmin.macro

```diff
@@ -72,6 +114,22 @@ definition {
 		Click(locator1 = "AppBuilder#SAVE_BUTTON");
 	}
 
+	macro selectForm {
+		Click(
+			key_ItemName = "${formName}",
+			locator1 = "AppBuilder#ITEM_ON_THE_TABLE_CLICK");
+
+		Button.clickNext();
+	}
+
```

Comment:
Hey Brooke, once you've created Deploy.macro, could you please move selectFormView and selectTableView macros to Deploy.macro.